book_source/book_intro/how_to_use_book.Rmd

```diff
@@ -38,11 +38,84 @@ tab at the bottom to see what we are up to. All you need is a github account and
 	If you do not wish your communication with the team to be public, send us an email at the 
 address above and we will get back to you as soon as possible.
 
-## How this Book is organized
-
-This book is organized into three main sections:
-
-1. A user section that is split three ways into a basic user's guide, advanced web interface guide, and an advanced user guide for using pecan from the command line. 
+# Main Topics
+
+This book is organized around these main topics:
+
+Basic User Guide
+	[Getting Started]
+	[The model-site page]
+	[Basic model configuration]
+	[VM Desktop]
+	[Conversion]
+Advanced User Guide - Web
+	[Advanced Web Interface Guide]
+	[Using Browndog]
+	[PEcAn XML Configuration]
+	[Parameter Data Assimilation]
+	[State Data Assimilation]
+	[Specific Model Configurations]
+	[Advanced Setup]
+Advanced User Guide - Cmd Line
+	[Advanced Command Line Guide]
+	[Run PEcAn from Command Line]
+	[Working with the VM]
+	[Extra Tools In PEcAn]
+User Appendix
+	[User Appendix]
+	[Updating PEcAn Code and Bety Database]
+	[PEcAn Models]
+	[Model Output Variables]
+	[Available Meteorological Drivers]
+PEcAn Workflow
+	[Workflow]
+	[The PEcAn Bety Connection]
+Development Guide
+	[Using Git]
+	[Directory Structure]
+	[Adding An Ecosystem Model]
+	[Met Data]
+	[How to Insert new Input Data]
+	[Coding Style]
+	[Roxygen2]
+	[Testing]
+	[Package Data]
+	[Logging]
+Appendix
+	[Installing PEcAn]
+	[Creating a Virtual Machine]
+	[CentOS/RedHat]
+	[CentOS/RHEL]
+	[Mac OSX]
+	[Ubuntu]
+	[Porting VM to AWS]
+	[Installing and configuring Shiny for PEcAn]
+	[FAQ]
```

Comment:
As a whole this sort of Table of Contents should be auto-generated rather than prescribed, as it creates a high potential for a disagreement between the side bar and the actual docs as people add/remove pages.